app/helpers/playbook/pb_doc_helper.rb

```diff
@@ -60,9 +60,9 @@ def get_kit_examples(kit, type)
     end
```

Comment:
While you are editing this file, could you get rid of the unneeded `return`s as well?